clustered/integration-test/src/test/java/org/ehcache/clustered/replication/OversizedCacheOpsPassiveTest.java

```diff
@@ -52,10 +52,12 @@
       newCluster(2).in(clusterPath())
         .withSystemProperty("ehcache.sync.data.gets.threshold", "2")
         .withServiceFragment(offheapResource("primary-server-resource", 2))
-        .withSystemProperty("JAVA_OPTS", "-Xms1024m -Xmx8192m")
+        .withServerHeap(2048)
+        .inline(false)
         .build();
 
   @Test
+  @Ignore("OOME on build slaves due to high memory requirements")
```

Comment:
Probably safer to `@Ignore` the class and leave the cluster setup unmodified for now.

Reply:
problem is `.withSystemProperty("JAVA_OPTS", "-Xms1024m -Xmx8192m")` does nothing the way galvan sets up the cluster